llvm/test/CodeGen/NVPTX/nvvm-reflect-module-flag.ll

```diff
@@ -4,6 +4,7 @@
 declare i32 @__nvvm_reflect(ptr)
 @str = private unnamed_addr addrspace(1) constant [11 x i8] c"__CUDA_FTZ\00"
 @str.1 = private unnamed_addr addrspace(1) constant [17 x i8] c"__CUDA_PREC_SQRT\00"
+@str.2 = private unnamed_addr addrspace(1) constant [16 x i8] c"__CUDA_PREC_DIV\00"
```

Comment:
I should've mentioned that in the other discussion, but the CUDA "libdevice" bitcode library doesn't use `PREC_DIV`, it only has ftz and prec sqrt, so for prec-div we should be able to skip all of that, and just use the regular offload flag, we only need this stuff for sqrt.